Nataly/src/components/previewMovies/previewMovies.component.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import "./previewMovies.styles.scss"
+import CardMovie from '../card-movie/card-movie.component';
+
+import {IPreviewMoviesProps, IMoviesDefault, IPreviewMoviesState} from "../../interfaces/interfaces"
+
+
+
+class PreviewMovies extends React.Component<IPreviewMoviesProps, IPreviewMoviesState,  IMoviesDefault> {
+    state = {
+        moreItem: false
+    } 
+
+    loadMore = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        console.log("load")
+            this.setState({
+                moreItem: true
+            })
+    }
+
+    render () {
+
+        const {moviesDefault} = this.props
+
+    return(
+    <div className="preview__movies-wrapper">
+       
+        <div className="preview__movies-card-wrapper">
+            { 
+            moviesDefault
+            .filter((item, index) => index < 9)
+            .map(({id, genres, title, release_date, ...item} : IMoviesDefault) => <CardMovie 
+            key={id} 
+            genres={genres}
+            title={title}
+            release_date={release_date}
+            {...item}
```

Comment:
Looks like you get tired of passing props and decided to pass all at ones. I don't think that you really need it.  And also when you pass props like this, you ruin type checking 
`<Component {...{...props, propThatNotInTypes: 'will get no error' }}>`

if you really need to pass like this, add "as"

`<Component {...{...props, propThatNotInTypes: 'will get error' } as ComponentType}>`